react-backend/client/src/components/authentication/pswdresetcb.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { Input } from '../visitors/input';
+import { Button } from '../visitors/button';
+import { Link } from 'react-router-dom';
+
+class CBPswdReset extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      error: []
+    };
+  }
+
+  handleChange = e => {
+    let newState = {};
+    newState[e.target.name] = e.target.value;
+    this.setState(newState);
+  };
+
+  setError(messagesArray) {
+    this.setState({ error: messagesArray });
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    const checkData = {
+      formEmail: this.state.email
+    };
+
+    fetch('/checkCBemail', {
```

Comment:
If you change the path in the router as noted above then you'll need to change it here too.